ptbcontrib/extbotcommand/README.md

```diff
@@ -0,0 +1,65 @@
+# Store a second, longer description
+
+Provides a class `ExtBotCommand` that allows you to store a second, longer description (>256 characters) for a `BotCommand` to be stored alongside a shorter description.
+Example:
+
+```python
+from typing import List
+
+from telegram import Update
+from telegram.ext import Updater, CommandHandler, CallbackContext
+from ptbcontrib.long_command_description import ExtBotCommand
```

Comment:
you used `extbotcommand`, though, not `long_command_description`.

Reply:
Done